json_chef.py

```diff
@@ -350,14 +350,14 @@ def convert_ka_node_to_ricecooker_node(ka_node, target_lang=None):
         files = [dict(file_type="video", path=download_url)]
 
         # include any subtitles that are available for this video
-        if target_lang not in UNSUBTITLED_LANGS:
+        if orig_target_lang.lower() not in UNSUBTITLED_LANGS:
             subtitle_languages = ka_node.get_subtitle_languages()
         else:
             subtitle_languages = []
 
         # if we dont have video in target lang or subtitle not available in target lang, return None
         if ka_node.lang != target_lang.lower():
-            if target_lang not in subtitle_languages:
+            if target_lang.lower() not in subtitle_languages:
```

Comment:
Here this list is from Youtube language codes, so better to leave the original `{lang}-{LOCALE}`